src/crewai/crew.py

```diff
@@ -623,6 +648,36 @@ def _execute_tasks(
                 )
             self._log_task_start(task, agent_to_use)
 
+            if isinstance(task, ConditionalTask):
+                if futures:
+                    task_outputs.extend(
+                        self._process_async_tasks(futures, was_replayed)
+                    )
+                    futures.clear()
+
+                previous_output = task_outputs[task_index - 1] if task_outputs else None
+                if previous_output is not None and not task.should_execute(
+                    previous_output
+                ):
+                    self._logger.log(
+                        "info",
+                        f"Skipping conditional task: {task.description}",
+                        color="yellow",
+                    )
+                    skipped_task_output = TaskOutput(
```

Comment:
A conditional_task should have a `get_skipped_output` or something like that so that the crew isn't responsible for generating this.